http_2.rflx

```diff
@@ -0,0 +1,230 @@
+package HTTP_2 is
+
+	type Payload_Length is mod 2**24; -- Must not be greater than 2**14 unless the receiver has set a larger value for SETTINGS_MAX_FRAME_SIZE
+
+	type Frame_Type is (
+		DATA => 0,
+		HEADERS => 1,
+		PRIORITY => 2,
+		RST_STREAM => 3,
+		SETTINGS => 4,
+		PUSH_PROMISE => 5,
+		PING => 6,
+		GOAWAY => 7,
+		WINDOW_UPDATE => 8,
+		CONTINUATION => 9
+	) with Size => 8;
+
+	type Settings_Parameter is (
+		SETTINGS_HEADER_TABLE_SIZE => 16#01#,
+		SETTINGS_ENABLE_PUSH => 16#02#,
+		SETTINGS_MAX_CONCURRENT_STREAMS => 16#03#,
+		SETTINGS_INITIAL_WINDOW_SIZE => 16#04#,
+		SETTINGS_MAX_FRAME_SIZE => 16#05#,
+		SETTINGS_MAX_HEADER_LIST_SIZE => 16#06#
+	) with Size => 32;
+
+	type Zero_Bit is range 0 .. 0 with Size => 1;
+
+
+	type Stream_ID is mod 2**31;
+	type Weight is mod 2**8; 		--RFC:Add one to the value to obtain a weight between 1 and 256?
+	type Error_Code is (
+		NO_ERROR => 16#00#,
+		PROTOCOL_ERROR => 16#01#,
+		INTERNAL_ERROR => 16#02#,
+		FLOW_CONTROL_ERROR => 16#03#,
+		SETTINGS_TIMEOUT => 16#04#,
+		STREAM_CLOSED => 16#05#,
+		FRAME_SIZE_ERROR => 16#06#,
+		REFUSED_STREAM => 16#07#,
+		CANCEL => 16#08#,
+		COMPRESSION_ERROR => 16#09#,
+		CONNECT_ERROR => 16#0A#,
+		ENHANCE_YOUR_CALM => 16#0B#,
+		INADEQUATE_SECURITY => 16#0C#,
+		HTTP_1_1_REQUIRED => 16#0D#
+	) with Size => 32;
+
+	type Settings_Value_Generic is mod 2**32;
+	type Settings_Value_Enable_Push is range 0 .. 1 with Size => 32;
+	type Settings_Value_Initial_Window_Size is range 0 .. 2**31 - 1 with Size => 32;
+	type Settings_Value_Max_Frame_Size is range 2**14 .. 2**24-1 with Size => 32;
+	type Window_Size_Increment is range 1 .. 2**31 - 1 with Size => 31;
+	type Pad_Length is mod 2**8;
+
+	type Frame is
+		message
+			Payload_Length : Payload_Length;
+			Frame_Type : Frame_Type;
+			Flag_End_Stream_ACK : Boolean
+				then Bit_2_Unused
+					if ((Frame_Type = SETTINGS and Payload_Length = 6) or
+						(Frame_Type = PING and Payload_Length = 8) or
+						(Frame_Type = DATA) or
+						(Frame_Type = HEADERS) or
+						((	Frame_Type = PUSH_PROMISE or
+							Frame_Type = CONTINUATION or
+							Frame_Type = GOAWAY or
+							Frame_Type = RST_STREAM or
+							(Frame_Type = WINDOW_UPDATE and Payload_Length = 4) or
+							(Frame_Type = PRIORITY and Payload_Length = 5)
+						) and Flag_End_Stream_ACK = False));
```

Comment:
If there are line breaks in an expression, the operator should be at the beginning of the next line:
```Ada
A
or B
or C
```

Reply:
Fixed in 4c1047c